lib/collection/src/shards/remote_shard.rs

```diff
@@ -156,7 +156,7 @@ impl RemoteShard {
     ) -> CollectionResult<UpdateResult> {
         // the target shard is None because the operation is forwarded as if it came from the client
```

Comment:
this comment needs to be changed now to reflect the new behaviour

Reply:
done